unity-renderer/Assets/DCLServices/WearablesCatalogService/WearablesCatalogServiceProxy.cs

```diff
@@ -0,0 +1,116 @@
+        {
+            kernelConfig.OnChange -= OnKernelConfigChanged;
+            wearablesCatalogServiceInUse?.Dispose();
+        }
+
+        public async UniTask<IReadOnlyList<WearableItem>> RequestOwnedWearablesAsync(string userId, int pageNumber, int pageSize, CancellationToken ct)
+        {
+            await UniTask.WaitUntil(() => isInitialized, cancellationToken: ct);
+            return await wearablesCatalogServiceInUse.RequestOwnedWearablesAsync(userId, pageNumber, pageSize, ct);
+        }
+
+        public async UniTask<IReadOnlyList<WearableItem>> RequestBaseWearablesAsync(CancellationToken ct)
+        {
+            await UniTask.WaitUntil(() => isInitialized, cancellationToken: ct);
+            return await wearablesCatalogServiceInUse.RequestBaseWearablesAsync(ct);
+        }
+
+        public async UniTask<IReadOnlyList<WearableItem>> RequestThirdPartyWearablesByCollectionAsync(string userId, string collectionId, CancellationToken ct)
+        {
+            await UniTask.WaitUntil(() => isInitialized, cancellationToken: ct);
+            return await wearablesCatalogServiceInUse.RequestThirdPartyWearablesByCollectionAsync(userId, collectionId, ct);
+        }
+
+        public async UniTask<IReadOnlyList<WearableItem>> RequestWearablesAsync(string[] wearableIds, CancellationToken ct)
+        {
+            await UniTask.WaitUntil(() => isInitialized, cancellationToken: ct);
+            return await wearablesCatalogServiceInUse.RequestWearablesAsync(wearableIds, ct);
+        }
+
+        public async UniTask<WearableItem> RequestWearableAsync(string wearableId, CancellationToken ct)
+        {
+            await UniTask.WaitUntil(() => isInitialized, cancellationToken: ct);
+            return await wearablesCatalogServiceInUse.RequestWearableAsync(wearableId, ct);
+        }
+
+        public void AddWearablesToCatalog(IReadOnlyList<WearableItem> wearableItems) =>
+            wearablesCatalogServiceInUse?.AddWearablesToCatalog(wearableItems);
+
+        public void RemoveWearablesFromCatalog(IEnumerable<string> wearableIds) =>
+            wearablesCatalogServiceInUse?.RemoveWearablesFromCatalog(wearableIds);
+
+        public void RemoveWearablesInUse(IEnumerable<string> wearablesInUseToRemove) =>
+            wearablesCatalogServiceInUse?.RemoveWearablesInUse(wearablesInUseToRemove);
+
+        public void EmbedWearables(IEnumerable<WearableItem> wearables) =>
+            wearablesCatalogServiceInUse?.EmbedWearables(wearables);
+
+        public void Clear() =>
+            wearablesCatalogServiceInUse?.Clear();
+
+        private void OnKernelConfigChanged(KernelConfigModel currentKernelConfig, KernelConfigModel previous)
+        {
+            kernelConfig.OnChange -= OnKernelConfigChanged;
+
+            if (currentKernelConfig.urlParamsForWearablesDebug)
+            {
+                WebInterfaceWearablesCatalogService.Instance.Initialize(new WearablesWebInterfaceBridge(), wearablesCatalog);
```

Comment:
i think that the `WearableWebInterfaceBridge` should be injected into the proxy

Reply:
Done!